### PR TITLE
Cal 28 types validation

### PR DESCRIPTION
**Pull request by RamiroMartinez2**

### CODE REVIEW

File: app/controllers/boilersTypes.js [hunk 1]
```diff
@@ -2,13 +2,26 @@ const db = require("../models");
...
+    ) {
+      res.status(400).send({ message: "Content can not be empty" });
+      return;
+    }
+    res.status(409).send({
+      message: ` ${req.body.stock} is not a number between 1 and 20 `,
+    });
+    return;
+  }
+
```
Comment (cocolobos) [inline app/controllers/boilersTypes.js:20]: delete white space

File: app/controllers/boilersTypes.js [hunk 2]
```diff
@@ -63,20 +76,29 @@ exports.findOne = (req, res) => {
...
+    return;
   }
-  if (
-    !req.body.id ||
-    !req.body.skillsId ||
-    !req.body.descriptions ||
-    !req.body.stock
-  ) {
-    res.status(400).send({ message: "Content can not be empty" });
+
```
Comment (cocolobos) [inline app/controllers/boilersTypes.js:94]: delete white space

**Review by cocolobos — COMMENTED** [review]
delete white spaces

**Review by lupoalejandro — APPROVED** [review]
OK

**Pull request merged**